docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotHashEncoding.cs

```diff
@@ -42,19 +43,25 @@ public static void Example()
             // 0 0 0 0 1 0 0 0
             // 0 0 0 0 0 0 0 1
 
-            // A pipeline for one hot hash encoding the 'Education' column (using keying strategy).
-            var keyPipeline = mlContext.Transforms.Categorical.OneHotHashEncoding("EducationOneHotHashEncoded", "Education", 
-                outputKind: OneHotEncodingEstimator.OutputKind.Key,
-                numberOfBits: 3);
+            // A pipeline for one hot hash encoding the 'Education' column
+            // (using keying strategy).
+            var keyPipeline = mlContext.Transforms.Categorical.OneHotHashEncoding(
+                "EducationOneHotHashEncoded", "Education",
+                OneHotEncodingEstimator.OutputKind.Key,
+                3);
 
             // Fit and transform the data.
-            var hashKeyEncodedData = keyPipeline.Fit(data).Transform(data);
+            IDataView hashKeyEncodedData = keyPipeline.Fit(data).Transform(data);
 
             // Getting the data of the newly created column, so we can preview it.
-            var keyEncodedColumn = hashKeyEncodedData.GetColumn<uint>("EducationOneHotHashEncoded");
+            var keyEncodedColumn = 
```

Comment:
>  [](start = 34, length = 1)

Extra trailing space. Is there a way to remove these automatically.

Reply:
- yes, via ctr+k, ctr-d

---
In reply to: [278757804](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278757804) [](ancestors = 278757804)